influxd.yaml

```diff
@@ -61,6 +61,54 @@ subpackages:
         - bash
         - yq
 
+  - name: ${{package.name}}-${{package.version}}-bitnami-compat
+    description: "compat package with bitnami/influxdb image"
+    pipeline:
```

Comment:
we'll require to add some runtime deps in here thinking of libgcc and more like `procps`, `bash`, `curl`, `coreutils`, `grep` and `sed` are required here if you look at the script files under script directories in the repo. 

```yaml
dependencies:
  runtime:
    - bash
    - coreutils
    - curl
    - grep
    - libgcc
    - procps
    - sed
```